databases/week2/cocoa_productionTABLE.sql

```diff
@@ -0,0 +1,40 @@
+
+CREATE TABLE cocoa_production (
+	`product_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	`product_name` VARCHAR(255) NOT NULL,
+	`start_date` DATETIME NOT NULL,
+	`due_date` DATETIME NULL DEFAULT NULL,
+	`product_location` FLOAT NOT NULL,
+    `price` DECIMAL(10,2) NOT NULL CHECK (price >= 0),
+     PRIMARY KEY (`product_id`,`product_name`),
+     FOREIGN KEY (`price`) REFERENCES customer(`order_id`)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+CREATE TABLE product_status (
+	`status_id`  INT(20) NOT NULL,
+	`status_name` VARCHAR(255) NOT NULL,
+     FOREIGN KEY (`status_name`) REFERENCES customer(`customer_id`)
```

Comment:
You are missing a PRIMARY KEY here. Also remember to think about your relations. If you want the products to have a status then where should the foreign key for the status be placed?